src/FlakeId/Extensions/IdExtensions.cs

```diff
@@ -8,14 +8,19 @@ public static class IdExtensions
         public static DateTimeOffset ToDateTimeOffset(this Id id) =>
             DateTimeOffset.FromUnixTimeMilliseconds(id.ToUnixTimeMilliseconds());
 
+        private const int TimestampOffset = Id.IncrementBits + Id.ProcessIdBits + Id.ThreadIdBits;
+        private const int ThreadOffset = Id.IncrementBits + Id.ProcessIdBits;
+        private const int ProcessOffset = Id.IncrementBits;
+
+

```

Comment:
nit: unnecessary newline

Reply:
Thanks! I didn't catch the extra empty line on my laptop.
I fixed it in da70313.